com.stansassets.google-doc-connector-pro/Runtime/Models/Spreadsheet.cs

```diff
@@ -48,6 +48,11 @@ public enum SyncState
         /// An event to track spreadsheet sync state.
         /// </summary>
         public event Action<Spreadsheet> OnSyncStateChange = delegate { };
+        
+        /// <summary> 
+        /// An event track the saving of spreadsheet data.
+        /// </summary>
+        public event Action OnDataStoredOnDisk = delegate { };
```

Comment:
```suggestion
        public event Action<DataStoredOnDiskArgs> OnDataStoredOnDisk = delegate { };
```
We should consider providing an Arguments struct for this `event Action` (yes, it will be just an empty `struct DataStoredOnDiskArgs{}` for the moment). It will save us from breaking changes in future updates. Just in case we will need to pass additional data with this `Action`.